lib/shared/addon/components/cluster-driver/driver-ociocne/component.js

```diff
@@ -270,25 +270,66 @@ export default Component.extend(ClusterDriver, {
 
     return out;
   }),
-  kubernetesVersionOptions: computed('config.region', 'primaryResource.cloudCredentialId', async function() {
+  ocneVersionOptions: computed('config.region', 'primaryResource.cloudCredentialId', async function() {
```

Comment:
Some of these have an 'Options' suffix and some don't - is there a reason?

Reply:
I did not put `options` suffix for the tags. If you'd like, I can add this

Reply:
No - was just curious